spec/services/calculators/multiples/multiple_offenses_calculator_spec.rb

```diff
@@ -56,6 +56,14 @@
       end
     end
 
+    context 'when there is an offence with `spent_simple`' do
+      let(:spent_dates) { [:spent_simple, :spent_simple] }
```

Comment:
Would be better to add a date to this array. The test in theory should behave identical to line 70, if there is one (or more) `spent_simple` those are considered spent, but then we use any other date in the array.